tests/libfuzzer/core_helper_fuzzer/core_helper_fuzzer.vcxproj

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
since we will eventually move to cmake based build, please add a cmakelists.txt file.

Reply:
Since the bpf2c_fuzzer and execution context fuzzer already got merged without one, I think all of them should be fixed at the same time as a separate issue

Reply:
Filed issue #1190